pyhf/commandline.py

```diff
@@ -28,6 +29,18 @@ def xml2json(entrypoint_xml, basedir, output_file, track_progress):
         json.dump(spec, open(output_file, 'w+'), indent=4, sort_keys=True)
         log.info("Written to {0:s}".format(output_file))
 
+@pyhf.command()
+@click.argument('workspace', default = '-')
+@click.argument('xmlfile', default = '-')
+@click.option('--specroot', default = click.Path(exists = True))
+@click.option('--dataroot', default = click.Path(exists = True))
```

Comment:
specroot and dataroot should just be config/data/results by default -- no? It's hard-coded for `HistFitter` and the goal is to at least be compatible with that (even though it's on top of HistFactory).

Reply:
in HistFactory it's usually `data` and `config`

Reply:
`results` is for the resulting workspace root file, normally.